### DISCUSSION TITLE
Creating individuals with genetic variation

### DISCUSSION THREAD

**Discussion by harrythmia** [Open-Systems-Pharmacology/Forum#988] [Questions & Problems] [answered]
Hello, 

I was just wondering is there any way for me to load a .fasta or .vca file into PK-Sim. I came across these genetic variants and decided to import them into PK-Sim. Such an example will be CYP2C9*1 variants from PharmVar (https://www.pharmvar.org/gene/CYP2C9). 

If those are not the accepted file type, may I know what will be the exact types of files PK-Sim accepts? 

Thank you very much.

Yours,
Harry

**Comment by DeniTue** [comment, marked as answer]
Dear Harry,

to my knowledge it is not possible to import genetic information like you asked for into PK-Sim, but rather the “effect” of different genetic variants, e.g., increased protein expression or decreased activity, can be modeled.

The current practice to model pharmacokinetics in different genetic variants is to create individuals and add the enzyme/transporter of interest as well as a respective process in the compound building block. When simulating your drug’s pharmacokinetics, the reference concentration can be changed in the individual or the activity, i.e., Km, vmax or kcat, can be changed in the compound according to literature (if available) or to parameter optimizations. If you want to model heterogenous allele carriers, you can add two versions of your enzyme/transporter of interest with halve the reference concentration each. Doing this, combinations with different expressions or activities "per allele" can be combined.

Here are some examples of drug-gene interaction modeling in PK-Sim: 
https://doi.org/10.3390/pharmaceutics12121200
https://doi.org/10.3390/pharmaceutics13030331

Best regards,
Denise

**Reply by harrythmia** [reply]
Dear Denise,

Thank you very much for clearing my doubt and provided me with greater knowledge about this software. 

Regards,
Harry
